mlflow/protos/databricks_uc_registry_messages.proto

```diff
@@ -299,7 +299,7 @@ message GenerateTemporaryModelVersionCredentialsRequest {
 
 message GenerateTemporaryModelVersionCredentialsResponse {
   // Temporary credentials for model version read/write
-  optional TemporaryCredentials credentials = 1 [(json_inline) = true];
```

Comment:
Removing this annotation because it's Databricks-specific and OSS doesn't know how to handle it